modules/react-native-chat/Components/Button.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+export default ({ title, onPress }) => {
+    return <TouchableOpacity onPress={onPress}>
+    <Text>{title}</Text>
+  </TouchableOpacity>;
+};
+export const InlineButton = ({ title, onPress }) => {
+    return <TouchableOpacity onPress={onPress} style={styles.btnStyle}>
+    <Text style={styles.txtStyle}>{title}</Text>
+  </TouchableOpacity>;
```

Comment:
```suggestion
    return (
      <TouchableOpacity onPress={onPress} style={styles.btnStyle}>
        <Text style={styles.txtStyle}>{title}</Text>
      </TouchableOpacity>
    );
```

Please use parentheses in JSX returns to increased readability and make the indentation more obvious.

Reply:
Resolved.